docs/pages/API/Button/index.js

```diff
@@ -0,0 +1,296 @@
+import React, { Component } from 'react'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { duotoneEarth } from 'react-syntax-highlighter/dist/styles/prism'
+import { Text } from '../../../../src'
+import ButtonChildrenSpecimen from '../../../specimens/ButtonChildren'
+import ButtonCustomizationsSpecimen from '../../../specimens/ButtonCustomizations'
+import ButtonModesSpecimen from '../../../specimens/ButtonModes'
+import ButtonSizesSpecimen from '../../../specimens/ButtonSizes'
+import ButtonStylesSpecimen from '../../../specimens/ButtonStyles'
+import style from '../style.less'
+
+const apiOptions = [{
+   name: 'borderRadius',
+   type: 'String',
+   default: '4px',
+   description: 'Set the border-radius of the button. By default, all buttons inherit their borderRadius from the <code>elementBorderRadius</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'className',
+   type: 'String',
+   default: '-',
+   description: 'Pass a <code>className</code> to customize className of the component',
+   responsive: false
+}, {
+   name: 'color',
+   type: 'String',
+   default: '-',
+   description: 'The primary button color (usually its background color). Pass any css compatible color value. By default, all buttons inherit their color from the <code>highlightColor</code> prop set on the parent <code>OIO</code> container.',
+   responsive: true
+}, {
+   name: 'fontFamily',
+   type: 'String',
+   default: '-',
+   description: 'Apply a specific font-family to the text within this component. By default, all buttons inherit the <code>fontFamily</code> set on the parent <code>OIO</code> container',
+   responsive: true
+}, {
+   name: 'mode',
+   type: 'Enum',
```

Comment:
I see you just started using this as a type. Since this is not actually a real thing in JS, we should probably discuss what this means and create a convention around it, and also update the other components listed in the docs to use `Enum` as well.